com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -210,6 +211,12 @@ public RenderTargetIdentifier cameraDepth
             get => m_CameraDepthTarget;
         }
 
+        public DebugMaterialIndex debugMaterialIndex { get; set; }
```

Comment:
We need to discuss how to encapsulate this for debug, this was very hacky solution for hackweek only.

Reply:
Done